app/controllers/claims/schools/users_controller.rb

```diff
@@ -1,3 +1,40 @@
 class Claims::Schools::UsersController < ApplicationController
   include Claims::BelongsToSchool
+
+  def index
+    @users = @school.users
```

Comment:
```suggestion
    @users = @school.users.claims
```

Reply:
Might be worth doing something like:
```
def users
  @users ||= @school.users.claims
end

Reply:
To add additional context here. The `Claims::School` redefines the `has_many :users` scope to include the `.claims` scope already.

This means, any `Claims::School#users` calls will already scope the users by the claims service.